src/walker.ts

```diff
@@ -293,6 +305,22 @@ export const flatten: Flatten = (obj, options = {}) => {
   return result
 }
 
+/**
+ * Unflatten an object previously flattened. Optionally pass `separator`
+ * to determine what character or RegExp to split keys with.
+ * Defaults to '.'.
+ */
+export const unflatten: Unflatten = (obj, options = {}) => {
```

Comment:
Besides the ability to override the separator, are there any differences in how this behaves compared to https://github.com/smartprocure/futil-js/blob/master/src/object.js#L179 ?

Reply:
That one only unflattens the top level whereas this one unflattens dotted paths at any level